server/server.js

```diff
@@ -15,7 +15,11 @@ const HEADERS = {
   "X-SESSION-TOKEN": process.env.HUB_AUTH_TOKEN,
 };
 
-const getEvents = (deviceUID) => {
+const getEvents = (deviceUID, timeframe) => {
+  if (timeframe === "undefined") {
+    timeframe = "8 days";
```

Comment:
Could you please add a comment about this magic value and make it a constant? 8 days isn't declared in `DATE_RANGE_OPTIONS` so the intent isn't totally clear.

Reply:
That's a good catch. We pull back 8 days' worth of data so the AQI averages can be calculated for the past 7 days, without including today's readings (which are still occurring).

But for the charts, I just need the last week up to and including today's readings as well, so I figured out a way to get both sets of readings in two different API calls to fetch the appropriate data